vcpkg.json

```diff
@@ -80,7 +80,7 @@
             "dependencies": [
                 {
                     "name": "onnx",
-                    "version>=": "1.15.0"
+                    "version>=": "1.16.0"
```

Comment:
it's not available yet https://vcpkg.link/ports/onnx

Reply:
yep, but it allows to prepare all other tasks while we are waiting